### ISSUE TITLE
Accessibility Testing for [Team Name, Product Name, Feature Name]

### ISSUE THREAD

**Issue by jasonday** [department-of-veterans-affairs/va.gov-team#123929]
### Product information

- [x] Team name, product name, and feature name have been added to the title of this issue.
- [x] Team label, product label and feature label (if applicable) have been added to this issue.

### Axe Devtools has been run on every page. (Required)

Pass

### Axe-core has been integrated in end to end testing. (Required)

Not tested

### Automated testing output

Axe results screenshots or link to ticket:

<img width="468" height="472" alt="Axe screenshot" src="https://github.com/user-attachments/assets/272fdb04-1c3b-4bd5-946a-bcd6aa7b5d0b" />

E2E testing results or link to ticket:

### Meaningful descriptions are provided for informative images (Required)

Pass

### No images of text (Required)

N/A

### Brief and detailed descriptions are provided for complex images (Recommended)

N/A

### Decorative images are hidden from screen readers (Recommended)

Pass

### Background images are not used for informative content (Recommended)

N/A

### Captions are provided for all prerecorded videos (Required)

N/A

### Transcripts or audio descriptions are included for videos (Required)

N/A

### Auto-playing audio can be paused or has volume controls (Required)

N/A

### Text transcripts are provided for audio and video-only content (Recommended)

N/A

### Real-time captions are provided for live videos (Recommended)

N/A

### Headings match the content hierarchy and use proper HTML tags (Required)

Pass

### Headings follow a logical order without skipping levels  (Required)

Pass

### There is one H1 per page/screen (Required)

Pass

### Each page has a unique, descriptive title (Required)

Fail

### Headings are descriptive (Required)

Pass

### Lists use proper list formatting. (Recommended)

Pass

### Content is organized into sections with headings. (Recommended)

Pass

### The page language is identified. (Recommended)

Pass

### Content in another language is identified. (Recommended)

N/A

### Instructions don't rely only on color, shape, size, or sound (Required)

Pass

### Information is not communicated by color alone (Required

Pass

### Text has sufficient contrast against its background (Required)

Pass

### Interactive elements are visually distinct from surroundings (Required)

Pass

### Important graphics and icons have sufficient contrast (Required)

Pass

### Text can be enlarged to 200% without breaking the page (Required)

Pass

### Content fits on small screens without horizontal scrolling (Required)

Pass

### Content works in both portrait and landscape mode. (Recommended)

Pass

### Text remains readable when spacing is adjusted. (Recommended)

Pass

### All functionality works with keyboard only (Required)

Pass

### No keyboard trap (Required)

Pass

### Every focusable element has a visible focus indicator (Required)

Pass

### Tab order follows a logical sequence. (Recommended)

Pass

### The element with focus is always visible. (Recommended)

Not tested

### Focusing on an element doesn't trigger unexpected changes. (Recommended)

Not tested

### Interacting with form fields doesn't trigger unexpected changes. (Recommended)

Not tested

### Automatically moving content can be paused or stopped (Required)

N/A

### Users can skip repeated content like headers and navigation (Required)

Pass

### Pages can be found in multiple ways. (Recommended)

Pass

### Navigation structure is the same across pages. (Recommended)

Pass

### Help options appear in the same location on all pages. (Recommended)

Pass

### Form labels clearly describe what to enter (Required)

Pass

### Form fields have visible labels (Required)

Pass

### Fields with specific formats include instructions (Required)

Pass

### Required or optional fields are clearly marked (Required)

Pass

### Error messages explain how to fix the problem (Required)

Pass

### Links navigate to pages; buttons perform actions (Required)

Pass

### Related form elements are grouped together. (Recommended)

Pass

### Required fields are identified programmatically (Recommended)

Pass

### Links are descriptive. (Recommended)

Pass

### Error messages are provided and are clear. (Recommended)

Not tested

### Add link to Collaboration Cycle ticket

- [x] Add a link to your completed accessibility testing ticket to the staging review artifacts section of your Collaboration Cycle ticket. This ticket can be closed after you finish your staging review.

### Report identified issues

- [x] Log each issue individually using the [Accessibility issue template](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?template=a11y-issue.yaml&assignees=jasonday&labels=a11y-testing&title=A11y+testing+finding%3A+%5Bshort+description%5D) filling in as much information as you are able.
- [x] Assign to `jasonday` and add the `a11y-testing` label.
- [x] Add the created issue to the Collaboration Cycle milestone found in your collab cycle ticket.
- [x] Provide a link to each issue logged in a comment on this ticket.
